packages/release-manifests/report.api.md

```diff
@@ -21,6 +21,12 @@ export function getManifestByVersion(
 // @public
 export type GetManifestByVersionOptions = {
   version: string;
+  fetchFn?: (
+    url: string,
+    options?: {
+      signal?: AbortSignal;
+    },
+  ) => Promise<Pick<Response, 'status' | 'json' | 'url'>>;
```

Comment:
Just flagging that we're potentially risking breaking changes here if in the future we need more of the properties of `Response` after fetching.